barcodes_search/wizards/barcode_search.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class BarcodeSearch(models.TransientModel):
+    _name = 'barcode.search'
+
+    barcode = fields.Char(
+        string='Barcode', required=True, help='Barcode to search for.')
+
+    state = fields.Selection(
+        selection=[('new', 'New'), ('done', 'Done')], default='new')
+
+    line_ids = fields.One2many(
+        comodel_name='barcode.search.line', inverse_name='barcode_search_id',
```

Comment:
Please define one per line as indicated in [idioms section for multiline lists](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms)